.github/workflows/src/user.js

```diff
@@ -0,0 +1,21 @@
+export default (sequelize, Datatypes) => {
```

Comment:
Please remove all .github workflows files that you have added, `availability.js`, `token.js`, `user.js` to avoid deployment issues as they are not required for your code to run.